paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerTest.kt

```diff
@@ -1748,58 +1746,30 @@ internal class DefaultFlowControllerTest {
     }
 
     @Test
-    fun `Launches CVC Recollection & succeeds payment`() = runTest {
-        fakeIntentConfirmationInterceptor.enqueueCompleteStep()
-
-        val onResult = argumentCaptor<ActivityResultCallback<CvcRecollectionResult>>()
-        val launcher = mock<CvcRecollectionLauncher> {
-            on { launch(any(), any(), any()) } doAnswer {
-                onResult.firstValue.onActivityResult(CvcRecollectionResult.Confirmed("123"))
-            }
-        }
-        val launcherFactory = mock<CvcRecollectionLauncherFactory> {
-            on { create(any()) } doReturn launcher
-        }
-
-        whenever(
-            activityResultCaller.registerForActivityResult(
-                any<CvcRecollectionContract>(),
-                onResult.capture()
-            )
-        ).thenReturn(mock())
-
-        val flowController = createFlowController(
-            cvcRecollectionLauncherFactory = launcherFactory,
+    fun `Launches CVC Recollection & succeeds payment`() {
```

Comment:
Do we have tests like this for PaymentSheet as well?

Reply:
There're tests in PaymentSheetViewModelTest and CvcRecollectionHandlerTest that cover launching CVC recollection successfully.